circle.yml

```diff
@@ -82,10 +78,57 @@ jobs:
             make install_system_dependencies
             sudo ln -s /usr/lib/x86_64-linux-gnu/libmysqlclient.so /usr/lib/x86_64-linux-gnu/libmysqlclient.so.18
             bin/install-supported-firefox
```

Comment:
@viadanna, this could be used to replace the Firefox installation code above or can be removed if it is duplicate work.

Reply:
:+1: